pyzeebe/channel/oauth_channel.py

```diff
@@ -0,0 +1,146 @@
+from functools import partial
+from typing import Optional
+
+import grpc
+from grpc.aio._typing import ChannelArgumentType  # type: ignore[import-untyped]
```

Comment:
1) We should not import from private modules of external libs.
2) It does not make sense to import smth for typing purposes and marking it as untyped.
3) ChannelArgumentType (tuple) != channel_options from other channel constructors.

We should use `Optional[Dict[str, Any]]` and `get_channel_options`.

Reply:
Commited the necessary changes to be compliant with linked grpc documentation.